src/helm/benchmark/runner.py

```diff
@@ -151,6 +152,15 @@ def __init__(
         exit_on_error: bool,
     ):
         self.executor = Executor(execution_spec)
+        self.annotator_executor = AnnotationExecutor(
+            AnnotationExecutionSpec(
+                local_path=execution_spec.local_path or "prod_env",
```

Comment:
Don't default to "prod_env"... if `local_path` is "", that means the local path is the present working directory.